qutebrowser/config/qtargs.py

```diff
@@ -203,12 +203,18 @@ def _qtwebengine_settings_args() -> Iterator[str]:
         }
     }
 
+    if qtutils.version_check('5.14', compiled=False):
+        # Starting in 5.15.2, the `--force-dark-mode` doesn't seem to toggle
+        # the media property, Instead, this will be handled by darkmode.py
+        # as it uses a blink-setting to force the value.
+        if not qtutils.version_check('5.15.2', compiled=False):
```

Comment:
@The-Compiler Should I just remove this? It doesn't hurt to have the flag there, even if it doesn't do anything in later versions. This also means I don't need to change the existing tests for this flag (which are currently failing.) Instead I'd only add new a new test to make sure the blink setting is correct in the darkmode-related tests.

Reply:
I'd rather get rid of `--force-dark-mode` and conditionals entirely if the new flag works on older versions too. If you want to give that a try, you can use e.g. `tox -e py38-pyqt514 --notest` to set up an environment; `.tox/py38-pyqt514/bin/python -m qutebrowser --temp-basedir -s content.... true` to start qutebrowser and e.g. `:jseval` to test.

If you prefer I can also take this over - I'd really like to get it finished (including a test verifying the behavior via the actual CSS selector in `test_invocations.py`) before the v1.14.1 release, probably early next week.

Reply:
If you write the CSS test I can remove the `--force-dark-mode` flag and try to add a test for `blink-settings` in `test_darkmode.py`.